packages/manager/modules/pci/src/projects/project/kubernetes/add/add.controller.js

```diff
@@ -124,9 +137,43 @@ export default class {
       quota: find(this.quotas, { region: this.cluster.region.name }),
     });
     this.loadFlavors(this.cluster.region.name);
+    this.loadPrivateNetworks();
     this.displaySelectedRegion = true;
   }
 
+  loadPrivateNetworks() {
+    this.availablePrivateNetworks = [
+      this.defaultPrivateNetwork,
+      ...sortBy(
+        map(
+          filter(this.privateNetworks, (network) => {
+            const region = find(network.regions, {
+              region: this.cluster.region.name,
+              status: 'ACTIVE',
+            });
+            set(network, 'clusterRegion', region);
+            return region;
+          }),
+          (privateNetwork) => ({
+            ...privateNetwork,
+            name: `${privateNetwork.vlanId.toString().padStart(4, '0')} - ${
+              privateNetwork.name
+            }`,
+          }),
+        ),
+        ['name'],
+      ),
+    ];
+    if (
+      this.cluster.privateNetwork &&
+      filter(this.availablePrivateNetworks, {
+        id: this.cluster.privateNetwork.id,
+      }).length === 0
```

Comment:
IMO we should use `some` here 
```suggestion
      !this.availablePrivateNetworks.some(({id}) => id === this.cluster.privateNetwork.id)
```

Reply:
I think find will be more appropriate as there will be only one entry

Reply:
The `find`/`some` implementations are the same : loop until find an item that match the criteria.
The only differences is that `find` returns the item, and `some` returns a boolean.
And here I think your are looking for a boolean (in a `if` condition).